### ISSUE TITLE
Google CSE XML is output without Title Element

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express_mono#806]
This is a "Contributed" module created by **University of Colorado Boulder.**

The Title element is not being populated although the code specifies that it should be:

https://github.com/CuBoulder/express_mono/blob/dev/modules/contrib/google_cse/templates/google_cse_export.tpl.php#L15

The output looks like: 

```
<?xml version="1.0" encoding="UTF-8" ?>
<CustomSearchEngine id="btj7w7xe9xi" creator="016005321760733004813" language="en" encoding="UTF-8" enable_suggest="true">
    <Title></Title> 
    <Context>

etc.
```

**Comment by cathysnider** [comment]
There may be other changes required to get our XML up to speed with Google Programmable Search.

**Comment by jwfuller** [comment]
@cathysnider To test to see if this is an actual issue

**Comment by cathysnider** [comment]
Digging into the whys and wherefores of this, I found that the Title element is populated from the 'Google Custom Search Engine Name' variable. That variable is set on a form that no Web Express person (including developers) has access to (here: admin/config/search/cse-settings). 

Instead, there is a specific form for Web Express sites here: admin/config/search/search-settings, which does not include the Google Custom Search Engine Name. We simply don’t allow site builders to choose their own CSE name. We input the title manually when we set up the Custom Search Engine on Google. 

Therefore, this whole thing is working as designed and the ticket should be closed.
